contrib/dropwizard/README.md

```diff
@@ -0,0 +1,119 @@
+# OpenCensus DropWizard Util for Java
+
+The *OpenCensus DropWizard Util for Java* provides an easy way to translate Dropwizard metrics to
+OpenCensus.
+
+## Quickstart
+
+### Prerequisites
+
+Assuming, you already have both the OpenCensus and Dropwizard client libraries setup and working
+inside your application.
+
+### Add the dependencies to your project
+
+For Maven add to your `pom.xml`:
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opencensus</groupId>
+    <artifactId>opencensus-contrib-dropwizard</artifactId>
+    <version>0.17.0</version>
+  </dependency>
+</dependencies>
+```
+
+For Gradle add to your dependencies:
+```gradle
+compile 'io.opencensus:opencensus-dropwizard:0.17.0'
+```
+
+### And the following code:
+
+```java
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.MetricRegistry;
+import io.opencensus.contrib.dropwizard.DropWizardMetrics;
+import io.opencensus.metrics.Metrics;
+
+public class YourClass {
+  // Create registry for Dropwizard metrics.
+  static final MetricRegistry codahaleRegistry = new MetricRegistry();
```

Comment:
use the full name for this because MetricRegistry is a common name (even OC has one).

Reply:
Made changes everywhere in README.md